src/asana/client.py

```diff
@@ -59,6 +59,31 @@ def create_task(self, project_id: str, due_date_str: str = None) -> str:
         response = self.asana_api_client.tasks.create(create_task_params)
         return response["gid"]
 
+    def create_subtask(
+        self, parent_task_id: str, reviewer: str, task_name: str, task_description,
```

Comment:
nit: I'd just call this `assignee` since this is a wrapper around the Asana API and doesn't necessarily need to know the context of "reviewer" - just to format and forward the request to the API

```suggestion
        self, parent_task_id: str, assignee: str, task_name: str, task_description,
```